deploy/nucleus-hub/operator.yaml

```diff
@@ -2,7 +2,7 @@ kind: Deployment
 apiVersion: apps/v1
 metadata:
   name: nucleus-hub
-  namespace: open-cluster-management-core
+  namespace: open-cluster-management
```

Comment:
I think we just tweaked this open-cluster-management-hub in the api repo here https://github.com/open-cluster-management/api/pull/25

Reply:
This namespace is for nucleus operator, it is not for hub core compoents,  the  hub core namespace should be open-cluster-management-hub